src/github.com/getlantern/flashlight/feed/feed.go

```diff
@@ -297,3 +289,20 @@ func determineLocale(defaultLocale string) string {
 	}
 	return defaultLocale
 }
+
+func getHTTPClient() (*http.Client, error) {
+	var err error
+	httpClientMutex.Lock()
+	if _httpClient == nil {
+		_httpClient, err = proxied.ChainedNonPersistent("")
```

Comment:
Awesome, @oxtoacart! I like the change to make the client global to take advantage of persistent connections. There is one issue though: I'm using the ``proxyAddr`` to indicate whether or not to use a proxied vs ordinary HTTP client. If none is specified, that means full-device VPN mode is enabled. (if it is enabled, we don't want to proxy any of those requests, otherwise those connections end up being rerouted back through the ``VpnService``). BTW, it would be useful to put this functionality in a separate package too (or maybe ``util``?) since its being duplicated in a few places now!